pyglet/clock.py

```diff
@@ -458,6 +458,28 @@ def schedule_interval(self, func, interval, *args, **kwargs):
         item = _ScheduledIntervalItem(func, interval, last_ts, next_ts, args, kwargs)
         _heappush(self._schedule_interval_items, item)
 
+    def schedule_interval_for_duration(self, func, interval, duration, *args, **kwargs):
+        """Schedule a function to be called every `interval` seconds
+        (see `schedule_interval`) and unschedule it after `duration` seconds.
+
+        The callback function prototype is the same as for `schedule`.
+
+        :Parameters:
+            `func` : callable
+                The function to call when the timer lapses.
+            `interval` : float
+                The number of seconds to wait between each call.
+            `duration=` : float
```

Comment:
Just a tiny typo here (=).

Reply:
ooops sorry, a remain of the keyword argument documentation...

thx